### PR TITLE
Reload Keybind & Cut (CTRL - X) Command.

### PR DESCRIPTION
**Pull request by sasial-dev**
In this commit is
Refresh keybind ([502850b](https://github.com/teverse/teverse/pull/61/commits/502850b0fd95a99226ca710ef23d070207bbd868))
And, a WIP Cut command ([3c4ed69](https://github.com/teverse/teverse/pull/61/commits/3c4ed698b5a29bac9983223b816eb54a10cd3339))
Refresh keybind was requested by another rpwb person and cut command I'll continue to work on once transparency is not just 0 and 1.

### CODE REVIEW

**Comment by sasial-roblox** [comment]
Most of the commits are just me updating from master.

**Comment by sasial-roblox** [comment]
Currently cut command set's the transparency of the current part(s) selected to 0 and then destroys it when pasted.
I'll need you to add something so that you can paste it where the mouse is as I don't believe that's possible with what I have.

File: workshop/controllers/core/clipboard.lua [hunk 1]
```diff
@@ -3,13 +3,33 @@ local history = require("tevgit:workshop/controllers/core/history.lua")
...
+local cut = false
 
 keybinder:bind({
     name = "copy",
     key = enums.key.c,
     priorKey = enums.key.leftCtrl,
-	action = function()
-		clipboard = selection.selection
+    action = function()
+        history.beginAction(workspace, "Copy")
```
Comment (teverse) [inline workshop/controllers/core/clipboard.lua]: You don’t need to begin action for copying; this will not achieve anything

Comment (sasial-roblox) [reply workshop/controllers/core/clipboard.lua]: Alright.

**Comment by sasial-roblox** [comment]
@teverse Was done yesterday.

**Comment by Sanjay-B** [comment]
Stale / old pull request. Closing.

**Pull request closed without merging**